pkgs/applications/networking/giara/default.nix

```diff
@@ -25,24 +27,25 @@ python3.pkgs.buildPythonApplication rec {
     owner = "World";
     repo = pname;
     rev = version;
-    sha256 = "004qmkfrgd37axv0b6hfh6v7nx4pvy987k5yv4bmlmkj9sbqm6f9";
+    hash = "sha256-xDIzgr8zYal0r0sASWqiSZANCMC52LrVmLjlnGAd2Mg=";
   };
 
   nativeBuildInputs = [
     meson
     gobject-introspection
     pkg-config
     ninja
-    wrapGAppsHook
+    wrapGAppsHook4
   ];
 
   buildInputs = [
-    gtk3
+    appstream
```

Comment:
Maybe appstream can move to nativeBuildInputs, I guess it is only used to validate metainfo file.

Reply:
Ok, thanks ... moved to nativeBuildInputs. Can you explain shortly how can you tell which Inputs to use? or point to documentation. Thanks

Reply:
Maybe check [nixpkgs-hammering](https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md)... `appstream` sometimes goes to `nativeBuildInputs` (to validate metainfo file) and sometimes goes to `buildInputs` (mostly for package management apps like `gnome-software`) I think.